CK2Plus_expanded/common/cultures/000_CK2Plus_amazigh.txt

```diff
@@ -438,4 +438,158 @@ amazigh = {
 
 		modifier = default_culture_modifier
 	}
+
+	houara = {

```

Comment:
tbh these are straight copies of zenati.

Reply:
If they're copies is there any reason to include them?

Reply:
we have no idea about their names, it is very hard to say to find distinguishing names among the berber tribes,

Reply:
Well, they're different cultures. Looking at the other cultures it seems like some of the others are fairly close to straight copies.